client/src/app/site/pages/meetings/pages/motions/view-models/view-motion.ts

```diff
@@ -222,7 +222,7 @@ export class ViewMotion extends BaseProjectableViewModel<Motion> {
 
     public get forwardingStatus(): ForwardingStatus {
         let status = ForwardingStatus.none;
-        if (!!this.all_origin_ids?.length) {
```

Comment:
I suspect you have to change the motion model subscription as well, or is the origin id already included there?